packages/block-editor/src/hooks/custom-fields.js

```diff
@@ -14,18 +14,18 @@ import { InspectorControls } from '../components';
 import { useBlockEditingMode } from '../components/block-editing-mode';
 
 /**
- * Filters registered block settings, extending attributes to include `__experimentalConnections`.
+ * Filters registered block settings, extending attributes to include `connections`.
  *
  * @param {Object} settings Original block settings.
  *
  * @return {Object} Filtered block settings.
  */
 function addAttribute( settings ) {
 	if ( hasBlockSupport( settings, '__experimentalConnections', true ) ) {
-		// Gracefully handle if settings.attributes is undefined.
+		// Gracefully handle if settings.attributes.connections is undefined.
 		settings.attributes = {
 			...settings.attributes,
-			__experimentalConnections: {
+			connections: {
```

Comment:
I'm not sure why these changes show up in GithHub's UI. They are already in the other PR: https://github.com/WordPress/gutenberg/pull/53241

Reply:
Maybe we need to rebase/merge the other PR branch. If I am not mistaken the last merge was done before the change you mention.

Reply:
yep, I had to merge `experiment/add-custom-connectinons-inspector-control` into this one before pushing again 🙂